src/main/java/TimeManager.java

```diff
@@ -0,0 +1,83 @@
+import java.time.DateTimeException;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * Contains utility functions for managing LocalDataTime object.
```

Comment:
This class is somehow confusing and hard to understand.
Here, "getTime" method actually allows user to input incomplete date and time information and then auto-complete using current date (if date/month/year is missing) or 1200am (if time is missing).
I will add comments and modification to make this more readable in later increments.